app/pulp/app/settings.py

```diff
@@ -184,7 +184,7 @@
         'working_directory': '/var/cache/pulp',
     },
     'broker': {
-        'url': 'qpid://localhost/',
+        'url': 'amqp://guest@localhost//',
```

Comment:
It's actually correct it signifies the default virtual host inside of rabbitMQ. Strange I know.

http://docs.celeryproject.org/en/3.1/getting-started/first-steps-with-celery.html#application